docs/Connection-Pools.md

```diff
@@ -0,0 +1,110 @@
+# Introduction
+
+SQLPP11 has support for connection pools which are centralized caches of database connections. When the user needs a database connection, he fetches one from the connection pool, uses the connection to make SQL
```

Comment:
That said, it might be more consistent to address the engineer directly as you do below, e.g.

```
When you need a database connection, you can fetch one from the connection pool, [...]
```

Reply:
OK, I changed the wording to address the user directly and expanded a bit the introductory part.

Reply:
Thanks!